back-end/API/tests/test_employee_jobOffer.py

```
@@ -0,0 +1,470 @@
+    "description": "Développeur fullstack",
+    "dateEntryOffice": "2021-12-12",
+    "deadlineApply": "2121-12-12",
+    "email": "test@gmail.com",
+    "hoursPerWeek": 40,
+    "offerLink": "www.google.com",
+    "salary": "1000",
+    "offerDebut": "2020-12-12",
+    "approbationMessage": "Super offre!",
+    "enterpriseId": 1,
+    "isApproved": True,
+    "approvedDate": datetime.now(),
```

Ruff (0.15.13)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)